src/commands.ts

```diff
@@ -1,24 +1,28 @@
-import { window, env } from 'vscode'
-import { getDescriptions } from './utils'
+import { window, env } from 'vscode';
+import { getDescriptions } from './utils';
 
 const copySpecificPath = (
-	command: keyof ReturnType<typeof getDescriptions>
+  command: keyof ReturnType<typeof getDescriptions>,
 ) => {
-	const editor = window.activeTextEditor
-	if (!editor) {
-		return
-	}
+  const editor = window.activeTextEditor;
+  if (!editor) {
+    return;
+  }
 
-	const descriptions = getDescriptions(editor)
-	env.clipboard.writeText(descriptions[command])
-}
+  const descriptions = getDescriptions(editor);
+  env.clipboard.writeText(descriptions[command]);
```

Comment:
Unfortunitely. I was not expecting PRs in the repo and have not set up linter/prettier rules.

For now please keep the styling as is. To make it easier to come up with a strict limiting in near future.